### ISSUE TITLE
flutter web plugin error ! 

### ISSUE THREAD

**Issue by metinemredonmez** [inspireui/support#4896]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
- [ ] first issue
- [ ] second issue
- [ ] third issue

6 warnings generated.
6 warnings generated.
ld: framework not found flutter_webview_plugin
clang: error: linker command failed with exit code 1 (use -v to see invocation)
note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description

please specificly problem !  how I'll solve this problem

+ Product & version: flutter full version 
+ Flutter (or React Native) version: 
+ Testing Device/Simulator: 
+ Screenshot issues (drag the file to attach here): 

<img width="858" alt="Ekran Resmi 2020-09-06 00 44 01" src="https://user-images.githubusercontent.com/54211029/92313933-1310d280-efda-11ea-8b4d-f9a9493d9409.png">

**_Step 2 (require): submit proof of purchasing the license on https://verify.inspireui.com **

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by minhcasi** [comment]
Duplicate of #4870
